Interfaces/About.html

```diff
@@ -0,0 +1,132 @@
+            <li class="list-item"><a  href="#">Program</a></li>
+            <li class="list-item"><a  href="#">Sponsor</a></li>
+            <li class="list-item"><a  href="#">New</a></li>
+            <li class="list-item"><a  href="#">Ticket</a></li>
+           </ul>
+           <div class="hambuger-container ">
+           <div class="bar"></div>
+           <div class="bar"></div>
+           <div class="bar"></div>
+           </div>
+          </nav>
+        </header>
+        <section class="landing-section1">
+          <div class="intro-section1">
+            <h2 class="openingText1">
+              Quarter mboko rappers           
+            </h2>
+            
+                 <h1 class="captureText1">
+                  About Mboko gods among us 2022
+                 </h1>
+            
+  
+                 <p class="explaining-text1">
+                    Mboko drill unearths Cameroon's hidden artists. It is a place where talent is discovered as others come to do what they do best in their careers. It began in Bamenda in 2015 and has since grown to become both national and international..
+                 </p>
+  
+                 <p class="eventDate1">Please contact us via email for any further question about mboko drill 2022</p>
+                 <a href="mailto:mboko@drill.com" class="venue1">mboko@drill.com</a>
+          </div>
+        </section>
+        
+        <section class="about-logo-container" >
+
+            <div class="text-time">
+           <h2  class="text-header2">Mboko drill gods among us 2022</h2>
+
+           <hr class="line2">
+            </div>
+
+            <div class="about-text">
+                <p>The logo of mboko drill gods among us was decided through the logo competition of from the last in 2020</p>
+            </div>
+
+            <div class="about-logo">
+                <img src="../Images/logo_image/logo-image.png" alt="logo-image">
+            </div>
+
+        </section>
+
+        <section class="recent-section">
+            <div class="text-time">
+                <h2 class="text-header2">See the past mboko drill event</h2>
+     
+                <hr class="line2">
+                 </div>
+     
+                 <div class="about-text2">
+                     <p>Take a look at the recent past two events about mboko drill gods among us.</p>
+                 </div>
+
+                 <div class="recent-carts">
+                    <div class="card card1">
+                        <h4>2020</h4>
+                        <p>mboko drill in Douala</p>
+                    </div>
+                    <div class="card card2">
+                        <h4>2018</h4>
+                        <p>mboko drill in Yaounde</p>
+                    </div>
+                 </div>
+        </section>
+
+
+        <section class="spnosor-section">
+            <div class="text-time">
+                 <h3  class="text-header">Spnosors</h3>
+                 <hr class="line">
```

Comment:
- [ ] [OPTIONAL] Great Job fixing the typo for the home page, kindly fix it for the about page as well, it should be `Sponsors`  instead of `Spnosors` 💯